src/styles/utils/variable.scss

```
@@ -0,0 +1,8 @@
+$color-main: #060b35;
+$color-secondary: #616070;
+$color-button-blue: #00acdc;
+$color-button-white: #fff;
```

Use more universal naming.

```suggestion
$color-button: #00acdc;
$color-button-active: #fff;
```